python/pylibcudf/pylibcudf/contiguous_split.pyx

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from cython.operator cimport dereference
+from libc.stdint cimport uint8_t, uintptr_t
+from libcpp.memory cimport make_unique, unique_ptr
+from libcpp.utility cimport move
+from pylibcudf.libcudf.contiguous_split cimport (
+    pack as cpp_pack,
+    packed_columns,
+    unpack as cpp_unpack,
+)
+from pylibcudf.libcudf.table.table cimport table
+from pylibcudf.libcudf.table.table_view cimport table_view
+
+from .gpumemoryview cimport gpumemoryview
+from .table cimport Table
+from .utils cimport int_to_void_ptr
+
+from types import SimpleNamespace
+
+import numpy as np
```

Comment:
Just noting that we'll need to explicitly include numpy as a required dependency for pylibcudf.